packages/integrations/node/test/bad-urls.test.js

```diff
@@ -41,6 +41,6 @@ describe('API routes', () => {
 		}
 		const stillWork = await fixture.fetch('/');
 		const text = await stillWork.text();
-		expect(text).to.equal('<!DOCTYPE html>\nHello!');
+		expect(text).to.equal('<!DOCTYPE html>Hello!');
```

Comment:
`compressHTML` is now true by default

Reply:
It's weird that I didn't catch this failing test before